core/App/ehsm_provider.cpp

```diff
@@ -938,7 +940,26 @@ ehsm_status_t GenerateQuote(ehsm_data_t *quote)
     return EH_OK;
 }
 
+/*
+ *  @brief get mrenclave and mrsigner from quote
+ *  @param quote_data quote data
+ *  @param mr_signer_str the mrsigner
+ *  @param mr_enclave_str the mrenclave
+ */
+void get_mrenclave_mrsigner_from_quote(const uint8_t* quote_data, 
+                                       string &mr_signer_str, 
```

Comment:
same here, why define it as string? rather than uint8_t[32]?

Reply:
copy understood.